hearthstone/entities.py

```diff
@@ -96,11 +99,21 @@ def __str__(self):
 	@property
 	def initial_deck(self):
 		for entity in self.entities:
-			if 3 < entity.id < 68:
-				if entity.tags.get(GameTag.CARDTYPE) not in (
-					CardType.HERO, CardType.HERO_POWER
-				):
-					yield entity
+			# Exclude entities created after setup
+			if entity.id not in self.game.initial_entities:
+				continue
+			# Exclude Player entities
+			if isinstance(entity, Player):
+				continue
+			# Exclude heros and hero powers
+			if entity.tags.get(GameTag.CARDTYPE) in (
+				CardType.HERO, CardType.HERO_POWER
```

Comment:
Additionally, if you check for the entity type instead, you don't have to check against player.